src/subapps/projects/containers/ActivityResourcesContainer.tsx

```diff
@@ -16,10 +16,20 @@ const ActivityResourcesContainer: React.FC<{
   linkCodeToActivity: (codeResourceId: string) => void;
 }> = ({ orgLabel, projectLabel, workflowStep, linkCodeToActivity }) => {
   const nexus = useNexusContext();
+  let base;
```

Comment:
maybe put this in a function `createWorkflowBaseFromContext` and add it in a lib, then you can create snapshot tests so that we can more easily read what the inputs and outputs look like